packages/components/src/DataViewer/Text/SimpleTextKeyValue/SimpleTextKeyValue.component.js

```diff
@@ -113,11 +113,21 @@ const SimpleTextKeyValue = React.forwardRef(function SimpleTextKeyValue(
 		style,
 		value,
 		displayTypes,
+		typesRenderer,
 		isValueOverflown,
 		isLongValueToggled,
 	},
 	ref,
 ) {
+	let types;
+	if (displayTypes && schema && value) {
+		types = (
+			<span className={classNames(theme['tc-simple-text-type'], 'tc-simple-text-type')}>
+				{typesRenderer ? typesRenderer(schema) : `- ${schema.type.type}`}
```

Comment:
Instead of an if, could you set a default value to `typesRenderer ` props please ?

Reply:
Done